<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSettingsWidget</class>
 <widget class="QWidget" name="AdvancedSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="shortcuts">
     <property name="title">
      <string>Keyboard Shortcuts:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="open">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Raise/Hide Last.fm</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ShortcutEdit" name="sce"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cache">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Cache Size:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="proxy">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Proxy:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="proxyType"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1" rowspan="2" colspan="3">
         <widget class="QLineEdit" name="proxyUsername"/>
        </item>
        <item row="4" column="1" colspan="3">
         <widget class="QLineEdit" name="proxyPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QLineEdit" name="proxyPort"/>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QLineEdit" name="proxyHost"/>
        </item>
        <item row="2" column="0" rowspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Host:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutEdit</class>
   <extends>QComboBox</extends>
   <header>../Widgets/ShortcutEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
